fabric-resource-loader-v0/src/client/java/net/fabricmc/fabric/mixin/resource/loader/client/VanillaResourcePackProviderMixin.java

```diff
@@ -23,16 +23,20 @@
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
-import net.minecraft.client.resource.ServerResourcePackProvider;
+import net.minecraft.client.resource.DefaultClientResourcePackProvider;
 import net.minecraft.resource.ResourcePackProfile;
+import net.minecraft.resource.VanillaResourcePackProvider;
 
 import net.fabricmc.fabric.impl.resource.loader.ModResourcePackCreator;
 
-@Mixin(ServerResourcePackProvider.class)
-public class ServerResourcePackProviderMixin {
+@Mixin(VanillaResourcePackProvider.class)
```

Comment:
What does this change?

Reply:
Fixes incorrect mixin injection point, `ServerResourcePackProvider` is used to provide HTTP downloaded packs. This is a better injection point for built in packs.